flowgger.toml

```diff
@@ -85,9 +85,19 @@ type = "file"
 file_path = "output.log"
 # Optional: Enables bufferized output. If rotation is used, must be smaller than file_rotation_size.
 file_buffer_size = 512
+
 # Optional: Enables file rotation once the specified size is reached.
-file_rotation_size = 980
-# Optional, only used if file_rotation_size is set: Specifies number of rotation files to use. The default value is 50.
+file_rotation_size = 2048
+
+# Optional: Enables file rotation based on time. Rotation occur every file_rotation_time minutes
+file_rotation_time = 2
+
+# Optional: When time rotation is enabled, format of the timestamp added to the filenames.
```

Comment:
Not very clear message. I would rephrase with

`Optional: When time rotation is enabled, the timestamp format is appended to the filenames.`

Reply:
alright